benefits/core/migrations/0002_data.py

```diff
@@ -282,7 +293,26 @@ def load_data(app, *args, **kwargs):
         enrollment_success_template="enrollment/success--sacrt.html",
     )
     sacrt_agency.eligibility_types.set([sacrt_senior_type])
-    sacrt_agency.eligibility_verifiers.set([sacrt_oauth_claims_verifier])
+    sacrt_agency.eligibility_verifiers.set([sacrt_senior_verifier])
+
+    sbmtd_agency = TransitAgency.objects.create(
+        slug="sbmtd",
+        short_name=os.environ.get("SBMTD_AGENCY_SHORT_NAME", "SBMTD (sample)"),
+        long_name=os.environ.get("SBMTD_AGENCY_LONG_NAME", "Santa Barbara MTD (sample)"),
+        agency_id="sbmtd",
+        merchant_id=os.environ.get("SBMTD_AGENCY_MERCHANT_ID", "sbmtd"),
+        info_url="https://sbmtd.gov/taptopride/",
+        phone="805-963-3366",
+        active=os.environ.get("SBMTD_AGENCY_ACTIVE", "True").lower() == "true",
+        private_key=client_private_key,
+        public_key=client_public_key,
+        jws_signing_alg=os.environ.get("SBMTD_AGENCY_JWS_SIGNING_ALG", "RS256"),
+        payment_processor=mst_payment_processor,
```

Comment:
Here I'm reused `mst_payment_processor` instead of defining a separate one for SBMTD because I didn't want us to have to set another set of `SBMTD_PAYMENT_PROCESSOR_...` secrets.

I was thinking it would be better if the data migration defined a QA `PaymentProcessor` and a Prod `PaymentProcessor`, and when we need to change which one a transit agency points at, we just change the object reference. I guess that would require a code change and deployment though...

Wondering what other people's thoughts are.

Reply:
I would rather we be consistent here with an SBMTD `PaymentProcessor` (and yes, all the secrets that entails... for now).

We don't want to necessarily advertise which agency is pointing to QA/Prod so don't like the idea of one for each. 

The Admin interface will make all of this moot.